### PR TITLE
main.py

### PR DESCRIPTION
**Pull request by erryte**
upload main.py

### CODE REVIEW

**Comment by heemonsu** [comment]
Hey @erryte, thanks for opening up the PR. However, you. do need to make a slight update.

The intention of the assignment is to make your PR in your own fork. This PR is attempting to fork into my repo, instead of your fork. 

To change this, you can:
1. Close this PR
2. Create a new PR from your fork, and make sure when opening the PR, you change the default target from heemosu:main (meaning the main branch of my fork) to your repo. 

Let me know if that makes sense!

File: main.py
Comment (heemonsu) [inline main.py:1]: @erryte, if you are writing a single file for execution, it usually a good idea to add a `def main()` function which defines your main execution, and then specifically define `if __name__ == '__main__'` where `main()` is called. 

This makes sure that the execution happens in a specific function scope, and not using any unexpected variables that are defined in the global scope unexpectedly. 

Let me know if that makes sense.

**Comment by erryte** [comment]
I made the change to main.py and opened a new PR. Can you see it or do I need to notify you again through GH?

**Comment by erryte** [comment]
incorrect PR

**Pull request closed without merging**